torch/csrc/distributed/c10d/NCCLUtils.hpp

```diff
@@ -12,6 +12,12 @@
 #include <c10/util/Exception.h>
 #include <c10/util/Optional.h>
 
+#ifndef NCCL_MAJOR
+  #ifndef NCCL_MINOR
+    #define NCCL_NOT_DEFINED
```

Comment:
where is it used?

Reply:
not used anymore, cleaning it up, thanks!